content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -10,17 +10,27 @@ weight: 10
 
 {{% capture overview %}}
 
-ReplicaSet is the next-generation Replication Controller. The only difference
+ReplicaSet is a type of Replication Controller. The only difference
 between a _ReplicaSet_ and a
-[_Replication Controller_](/docs/concepts/workloads/controllers/replicationcontroller/) right now is
+[_ReplicationController_](/docs/concepts/workloads/controllers/replicationcontroller/) right now is
 the selector support. ReplicaSet supports the new set-based selector requirements
 as described in the [labels user guide](/docs/concepts/overview/working-with-objects/labels/#label-selectors)
 whereas a Replication Controller only supports equality-based selector requirements.
 
+
 {{% /capture %}}
 
 {{% capture body %}}
 
+## How a ReplicaSet Works
+
+Just like a ReplicationController, a ReplicaSet's purpose is to maintain a stable set of replica Pods available at any
+given time. It does so by creating and deleting a pods as needed to reach the desired number. The link a ReplicaSet has
```

Comment:
Our style is to capitalize the names of Kubernetes objects. So Pod should be capitalized throughout the topic.

Reply:
fixed